Formula/gnu-complexity.rb

```diff
@@ -21,12 +21,25 @@ class GnuComplexity < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "6bc40505bf964f2ac7ef30d2f65c8180832e709c49cf6872b8651caf6a84b1a1"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "autogen"
 
+  # Fix build problem in doc. Borrowed from Debian.
```

Comment:
Has this been upstreamed?

Reply:
No. `master` and `v1.13` are at the same commit: https://git.savannah.nongnu.org/cgit/complexity.git

Reply:
Could you submit the fix upstream then?

Reply:
Done. Reported both issues to project maintainer @brkorb via email, as I couldn't find another way.

Reply:
The author/maintainer has no idea how to incorporate this into "upstream".
gnu-complexity.rb is not part of the sources. :)

Reply:
It's about this patch though: https://salsa.debian.org/debian/complexity/-/raw/69a7b9d27eb5c2ba8aa43966518971df74d55657/debian/patches/01_fix_autobuild.patch

Reply:
If you're improving the patches in some way, please keep me in the loop